src/Morphir/Spark/AST.elm

```diff
@@ -67,12 +67,55 @@ These are the supported Transformations:
           - A List of (Name, Expression) which represent a alias for a column expression and a column expression
             like `col(name)` within spark.
           - A target ObjectExpression from which to select.
+  - **Aggregate**
+      - Represents a `df.groupBy(...).agg(...)` transformation.
+      - The arguments are:
+          - A String, which represents the column to group aggregations by.
```

Comment:
I'm not sure if we explicitly mentioned this in any task but we should support grouping by multiple columns. Let me know if that's the case and I'll add a specific task for it.

Reply:
I don't think it was ever discussed, but my understanding of how Morphir.SDK.Aggregate.groupBy works from its examples was that groupBy only takes a single FieldFunction to perform the grouping, so it can only be grouped by one column.

Reply:
If you can show me an example of how to group multiple columns using the Morphir.SDK.Aggregate API I can implement it, though perhaps in a new PR.

Reply:
You reasoning is sound. The trick is that we use tuples to represent composite keys. There's only one example of that in the `Aggregation` module: https://github.com/finos/morphir-elm/blob/118e7be79fb47b6d51baf53dec2a54ff1c7ab4bb/src/Morphir/SDK/Aggregate.elm#L289

Here, `key2` is a utility function that simply composes two individual field functions into one that extracts both fields as a tuple. We have many `keyN` functions where `N` is the number of fields in the composite key.

Reply:
Thanks! I'll write up a new issue describing what needs to be done to extend aggregations to handle multiple columns.